src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECK_STATUS = 'CHECK_STATUS';
+
+const checkStatusReducer = (state = [], action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return action.payload;
+
+    default:
+      return state;
+  }
+};
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+  payload: 'Check Status',
```

Comment:
**OPTIONAL** - The string returned should be "Under Construction". Please fix this 👍